packages/pdfx/lib/src/viewer/pinch/pdf_view_pinch.dart

```diff
@@ -548,7 +550,8 @@ class _PdfViewPinchState extends State<PdfViewPinch>
           Rect.fromLTWH(-m.row0[3], -m.row1[3], viewSize.width, viewSize.height)
               .inflate(_padding);
 
-      for (final page in _pages) {
+      for (var i = 0; i < _pages.length; i++) {

```

Comment:
For what?

Reply:
When I open a pdf and close it quickly, it throws an error that this for is iterating a list that is empty, because when the page gets disposed, the _pages list gets cleared.